index.html

```diff
@@ -529,15 +529,26 @@ <h2>
 
     <section>
       <h2>
-Keys
+Cryptographic Keys
       </h2>
       <p>
-One or more <a href="#public-keys"></a> can be included in a <a>DID
-Document</a> using, for example, the <code>publicKey</code> or <code>authentication</code>
-properties depending on what they are to be used for. Each public key
-has an identifier (<code>id</code>) of its own, a <code>type</code>,
-and a <code>controller</code>, as well as other properties which
-depend on what type of key it is.
+A DID Document can express cryptographic keys and other verification methods
+that can be used to authenticate or authorize interactions with the subject of
+the DID or associated parties. The information expressed often includes globally
+unambiguous identifiers and public key material which can be used to verify
+digital signatures. Other information can be expressed such as status
+information for the key, such as if it has been suspended or revoked, or other
+attributes that enable one to verify if it is a hardware-backed cryptographic
```

Comment:
```suggestion
attributes that enable one to verify that it is a hardware-backed cryptographic
```

Reply:
(More pedantically, it should be either "determine if" or "verify that")

Reply:
"determine whether" ;-)

Reply:
Applied @TallTed's language.